docs/sovereign/setup.md

```diff
@@ -13,23 +15,19 @@ This guide will help you deploy contract on main chain, set up configuration fil
     mxpy wallet new --format pem --outfile ~/wallet.pem
     ```
 
+2. Get funds in the wallet on the chain you want the sovereign to be connected to.
+
 :::note
 You can use any wallet of your choice, but for the purpose of this guide we are taking the step by step approach as if there is nothing available nor installed on your node.
 :::
 
-## Step 2: Download necessary contracts
-
-2. Download the WASM files from [here](https://github.com/multiversx/mx-sovereign-sc/releases).
-
-3. For the purpose of the automated setup and deployment, save them in a directory called `contracts` in the same location where the wallet has been stored. Otherwise you could configure their own location at step number 4.
+## Step 2: Get the ```mx-chain-go``` Repository
 
-## Step 3: Clone the ```mx-chain-go``` Repository
+Before proceeding, ensure that a **SSH key** for GitHub is configured on your machine.
```

Comment:
If you changed all git cones without SSH(change below without https), I don't think this is required?

Reply:
I changed all `git clone` commands to use SSH